pygef/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.4"
+__version__ = "0.4.1b"
```

Comment:
Could you drop the version bumps. This should not be done at commit level.

Reply:
I would prefer not to, I did this merge request in order to have the same repository in gitlab and gihub. We needed to bump the version in gitlab because if you don't bump the version and make changes in the docstrings they are not updated in the documentation (Sphinx). Furthermore, we just added a patch (1b) that was not present before.

Reply:
You can refer commit hashes for that. ;)